app/src/main/java/com/taskmuse/app/ui/fragment/authentication/SignupFragment.java

```diff
@@ -28,6 +28,7 @@
 import com.google.firebase.auth.FirebaseUser;
 import com.taskmuse.app.R;
 import com.taskmuse.app.ui.activity.MainActivity.MainActivity;
+import com.taskmuse.app.ui.fragment.get_started.GetStartedFragment;
```

Comment:
Is this import being used as I can only see the main activity function being called in this file.

Reply:
I have removed this file in the newest commit, along with others that were unused in this file.